test/unit/math/prim/mat/vectorize/foo_base_test.hpp

```diff
@@ -0,0 +1,92 @@
+#ifndef TEST_UNIT_MATH_PRIM_MAT_VECTORIZE_FOO_BASE_TEST_HPP
```

Comment:
Can we just write an example test instead of `foo_base_test`? Perhaps just write `exp_base_test.hpp`?